### PR TITLE
Issue/132

### LINKED ISSUE #132: Scratch blocks to work with other codeblocks
Caused by issues with the `fenced_code` extension

### PR DESCRIPTION
**Pull request by ravenmaster001**
fixes #132 

Please give this a thorough look over.

### CODE REVIEW

File: docs/source/processors/scratch.rst [hunk 1]
```diff
@@ -25,6 +31,11 @@ You can test the output of your Scratch block text at
     (for example: Django) to create the images.
     See :ref:`accessing-scratch-image-data` section below.
 
+.. note::
+
+    The following examples assume usage of the fenced code extension, by having
```
Comment (JackMorganNZ) [inline docs/source/processors/scratch.rst]: Is this true? As I thought the extension could work without fenced code extension.

Comment (ravenmaster001) [inline docs/source/processors/scratch.rst]: Yes all the examples we show have to uses the fenced code extension. It does work without the extension though see the test: example_standard_markdown_block.

Comment (JackMorganNZ) [inline docs/source/processors/scratch.rst]: Thanks for the clarification.

Comment (JackMorganNZ) [inline docs/source/processors/scratch.rst]: This needs to move to the top of the file (probably line 6) as the first example uses fenced code example.

**Pull request merged**